util/git/client.go

```diff
@@ -353,17 +376,15 @@ func (m *nativeGitClient) LsRemote(revision string) (string, error) {
 	// symbolic reference (like HEAD), in which case we will resolve it from the refToHash map
 	refToResolve := ""
 	for _, ref := range refs {
-		refName := ref.Name().String()
-		if refName != "HEAD" && !strings.HasPrefix(refName, "refs/heads/") && !strings.HasPrefix(refName, "refs/tags/") {
-			// ignore things like 'refs/pull/' 'refs/reviewable'
+		shortRefName, ok := getShortRefName(remote, ref.Name())
```

Comment:
This is indirectly covered via `TestLsRemoteWithFetchRefspec` in util/git/git_test.go. There, the refspec `+refs/pull/*/head:refs/remotes/origin/pull-requests/*` is tested, so that it maps `refs/pull/1/head` to the short name `pull-requests/1`. Is that sufficient? What other tests would you like to see?

I'm pondering about what to do if there are multiple refs that would match a given short name. Say a tag and a branch with the same name, for example. Then I could add tests to verify the expected behavior.